core/trino-main/src/test/java/io/trino/block/BlockAssertions.java

```diff
@@ -329,6 +351,15 @@ public static Block createEmptyLongsBlock()
         return BIGINT.createFixedSizeBlockBuilder(0).build();
     }
 
+    public static Block createAllNullsBlock(Type type, int positionCount)
```

Comment:
`BlockAssertions` class should contain assertions. Put the rest in some `BlockTestuUtils` or something.

Reply:
This class contains both assertions and helper methods since its birth. Right now it actually has only one assertion so it may be better to just rename this class to BlockTestuUtils. 
I do feel it should not be a part of this though.